src/theme/mediaQueries.ts

```diff
@@ -0,0 +1,47 @@
+import { lg, md, sm, xl } from './breakpoints'
+
+export const mediaQueries = {
+  /**
+   * Min with media queries
+   */
+  sm: `@media screen and (min-width: ${sm}px)`,
+  md: `@media screen and (min-width: ${md}px)`,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'md' is not sorted alphabetically](https://app.codacy.com/app/PrideInLondon/pride-london-web/pullRequest?prid=4694377)